log-manager/src/main/java/io/airlift/log/JsonRecord.java

```diff
@@ -0,0 +1,99 @@
+package io.airlift.log;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Throwables;
+
+import java.time.Instant;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class JsonRecord
+{
+    private final Instant timestamp;
+    private final Level level;
+    private final String thread;
+    private final String loggerName;
+    private final String message;
+    private final Throwable thrown;
```

Comment:
Should we call this (and the getters) `throwable`? This better matches the logger API which logs a throwable, although `java.util.logging.Logger` and `LogRecord` calls it `thrown`.